src/Domain/Operation/isOperationPermitted.ts

```diff
@@ -0,0 +1,22 @@
+import { UserPermissions } from '../User/UserWithPermissions'
+import { Operation } from './Operation'
+import { Resource } from '../Resource/Resource'
+
+// todo: implement based on what auth is capable of returning; + tests
```

Comment:
Don't leave to-do's in code. They are most of the time left over and forgotten. Please create issues/tickets in project management tools instead.

Reply:
How about a rule like this: when creating a PR todo comments are allowed, but before merging they must be either resolved or turned into tasks.

This is close to my usual workflow and I find it pretty effective. I do my best to maintain my (todo or other) comments.

The disadvantages of tickets, especially for small tasks are that they are more difficult for me to create, find, relate to exact lines of code they pertain to, and they are much less practical for reviewers. I'm not sure if they would be easier to maintain.

I find todo comments to be the simplest and easiest way to keep track of and remember stuff that needs to be done in code.

What do you say?

BTW, about this comment: any idea how and when to implement this? Do we know already what exactly does/will `auth` return?

Reply:
I think the issue is as Karol said: "They are most of the time left over and forgotten." They're not a problem until they're forgotten. But I did notice you cleaned them up pretty nicely towards the end.

Reply:
Created a task for these todo comments: https://app.asana.com/0/1199914526147392/1200093443903077/f

Reply:
Removed all todo comments. Resolving on the account of that.